server/utils/nunjucksSetup.ts

```diff
@@ -23,6 +23,7 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
   app.locals.paths = paths
   app.locals.dateFormats = DateFormats
   app.locals.googleAnalyticsId = config.analytics.googleAnalyticsId
+  app.locals.features = config.features
```

Comment:
This makes feature flags available in Nunjucks (e.g. you can do `{% if features.myFlag %}...`)